pkg/api/mlflow/config/config.go

```diff
@@ -34,8 +34,8 @@ func NewServiceConfig() *ServiceConfig {
 		ListenAddress:         viper.GetString("listen-address"),
 		AuthUsername:          viper.GetString("auth-username"),
 		AuthPassword:          viper.GetString("auth-password"),
-		ArtifactRoot:          viper.GetString("artifact-root"),
 		S3EndpointURI:         viper.GetString("s3-endpoint-uri"),
+		DefaultArtifactRoot:   viper.GetString("default-artifact-root"),
```

Comment:
if you move it down, maybe sort the list?

Reply:
I though about that too =) but since it is a bit related to `s3-endpoint`, decided to keep it here.

Reply:
but ok, let me move it.